lib/core/test/unit/Cardano/Wallet/Primitive/TypesSpec.hs

```diff
@@ -228,6 +230,14 @@ spec = do
                 toText <$> fromText @(Hash "Account") text
                     `shouldBe` Right text
 
+        it "Can roundtrip {decode,encode}PoolIdBech32" $
+            withMaxSuccess 1000 $ property $ \(pid :: PoolId) ->
+                let enc = encodePoolIdBech32 pid
+                    dec = decodePoolIdBech32 enc
+                in case dec of
+                     Left _ -> False
+                     Right dec' -> dec' == pid
```

Comment:
4-space indents:

```suggestion
                in case dec of
                    Left _ -> False
                    Right dec' -> dec' == pid
```